modules/data.remote/R/NLCD.R

```diff
@@ -38,7 +38,7 @@ download.NLCD <- function(outdir, year = 2011, con = NULL) {
     ## Download the data
     dir.create(outdir, showWarnings = FALSE)
     destfile <- file.path(outdir, paste0("nlcd", year, ".zip"))
-    download.file(url, destfile = destfile)
+    PEcAn.utils::download_file(url, destfile = destfile)
```

Comment:
Please convert this to `utils::download.file` -- this was (and should remain) a call to the base R `download.file` and the updated call will fail because it doesn't know `destfile` is the same argument it calls `filename`.